tests/cli/test_events.py

```diff
@@ -161,3 +161,162 @@ async def test_stream_oss_events(
         assert parsed_event.event == example_event_1.event
     except ValueError as e:
         pytest.fail(f"Failed to parse event: {e}")
+
+
+async def test_emit_event_simple():
+    await run_sync_in_worker_thread(
+        invoke_and_assert,
+        [
+            "events",
+            "emit",
+            "user.action",
+            "--resource-id",
+            "user-123",
+        ],
+        expected_code=0,
+        expected_output_contains=[
+            "Successfully emitted event 'user.action'",
+        ],
+    )
```

Comment:
Can you update the tests that successfully emit an event to read the emitted event back to ensure it has the expected shape?